.github/workflows/pull-request.yml

```diff
@@ -1,94 +0,0 @@
-name: Pull Request
-
-on:
-  push:
-    branches: '**'
-
-defaults:
-  run:
-    shell: bash
-
-jobs:
-  clippy:
```

Comment:
Whats the reasoning behind no clippy?

Reply:
`yarn run lint:check` runs every `lint:check` script in workspaces and clippy will run in `yarn run lint:check` in programs workspace.